utils/dc-chain/scripts/clean.mk

```diff
@@ -21,8 +21,9 @@ clean_patches_stamp:
 clean: clean_patches_stamp
 	-rm -rf build-newlib-$(sh_target)-$(newlib_ver)
 	-rm -rf build-newlib-$(arm_target)-$(newlib_ver)
-	-rm -rf build-gcc-$(sh_target)-$(sh_gcc_ver)
-	-rm -rf build-gcc-$(arm_target)-$(arm_gcc_ver)
+	-rm -rf build-gcc-$(sh_target)-$(sh_gcc_ver)-pass1
+	-rm -rf build-gcc-$(sh_target)-$(sh_gcc_ver)-pass2
+	-rm -rf build-gcc-$(arm_target)-$(arm_gcc_ver)-pass1
```

Comment:
Should the arm build have -pass1 on it if there's only one pass to begin with?

Reply:
Today it's the case indeed but I don't think it's necessary

Reply:
Yes indeed I think it would be OK like that 👍🏻